runner/aws_lambda_scripts/invoke_lambda.sh

```diff
@@ -0,0 +1,69 @@
+# MIT License
+#
+# Copyright (c) 2022 Alan Nair and The vHive Ecosystem
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+
+#!/bin/bash
+set -e
+
+# DEPLOY LAMBDA FUNCTION FROM ECR CONTAINER
+
+#######################
+# WORKING DIRECTORY is vSwarm ROOT directory (../..)
+# All paths are relative to WORKING DIRECTORY
+#
+# On platforms such as github workflows, Working Directory is already set
+# and awscli is already installed
+# On such platforms, env var WORKDIR_SET must be set to some value.
+#
+# On other platforms (eg. if you are running locally), first we must
+# cd to WORKING DIRECTORY and install awscli
+# Make sure that env var WORKDIR_SET is not set.
+# see below.
+if [[ -z "${WORKDIR_SET}" ]]; then
+	cd ../..
+	pip install -U awscli
+fi
+#######################
+
+AWS_DEFAULT_REGION=${AWS_DEFAULT_REGION:-'us-west-1'}
+
+function sanitize() {
```

Comment:
Please add a comment explaining this as bash is non-intuitive :smile: .

Reply:
You mean this: `VARIABLE=${VARIABLE:-'default_value'}` ?
It means if $VARIABLE is not set (equal to "") then give it a default value.
It has already been used in several other scripts in this repository ([this one](https://github.com/vhive-serverless/vSwarm/blob/aws_lambda_support/runner/scripts/01-kind.sh#L7), [this one](https://github.com/vhive-serverless/vSwarm/blob/aws_lambda_support/runner/scripts/02-contour.sh#L6), [this one](https://github.com/vhive-serverless/vSwarm/blob/aws_lambda_support/runner/scripts/02-kourier.sh#L6), [this](https://github.com/vhive-serverless/vSwarm/blob/aws_lambda_support/runner/scripts/02-serving.sh#L6) and so on).

Reply:
Got it. Thanks.
Btw those scripts are borrowed from somewhere too :smile: .